correctness/all.js

```diff
@@ -0,0 +1,73 @@
+        "@typescript-eslint/explicit-module-boundary-types": [                                              // Enforce return types on exported methods
+            "error",
+            { "allowArgumentsExplicitlyTypedAsAny": true },
+        ],
+        "@typescript-eslint/member-delimiter-style": [                                                      // Require ";" after member declarations
+            "error",                                                                                        // Last ";" is not required on single line
+            {
+                "multiline": {
+                    "delimiter": "semi",
+                    "requireLast": true,
+                },
+                "singleline": {
+                    "delimiter": "semi",
+                    "requireLast": false,
+                }
+            }
+        ],
+        "@typescript-eslint/no-empty-function": "off",                                                      // Disallow empty functions, including callbacks
+        "@typescript-eslint/no-explicit-any": "error",                                                      // Disallow use of the "any" type
+        "@typescript-eslint/no-unnecessary-condition": "error",                                             // Dissallows unnecessary "?.", "if (0 === 0)" etc
+        "@typescript-eslint/strict-boolean-expressions": "error",                                           // Disallows coercing non-booleans to boolean
+        "@typescript-eslint/no-non-null-assertion": "error",                                                // Disallows usage of "!" e.g: lUndefined!;
+        "no-unused-vars": "off",                                                                            // turn off eslint variant for correct @typescript output
+        "@typescript-eslint/no-unused-vars": ["error", { "ignoreRestSiblings": true, "varsIgnorePattern": "^__" }],     // Disallows unused vars, unless __Type;
+        "@typescript-eslint/prefer-nullish-coalescing": "error",                                            // Prefer "??" over "||", so we don't do a falsy check
+        "@typescript-eslint/prefer-readonly": "error",                                                      // Prefer readonly private members where possible
+        "@typescript-eslint/quotes": ["error", "double", { "allowTemplateLiterals": true }],                // Use "" quotes instead of '', or ``
+        "@typescript-eslint/type-annotation-spacing": "error",                                              // const foo: number, space after ":"
+        "@typescript-eslint/switch-exhaustiveness-check": "error",                                          // Requires all branches to be covered or a default
+        "@typescript-eslint/tslint/config": [                                                               // Legacy tslint rules, runs our semicolon rules
```

Comment:
We should move this to `style/standard` because some (but not all) of the rules will affect styling